mpd/README.md

```diff
@@ -3,296 +3,518 @@
+            next:
+                type: bool
+                mpd_command@kitchen: next
+                enforce_updates: true
 
-### add(file)
-Adding the specified file, directory or URL to the playlist.
+            previous :
+                type: bool
+                mpd_command@kitchen: previous
+                enforce_updates: true
 
-## Example item.conf (deprecated) / item.yaml
+            stop :
+                type: bool
+                mpd_command@kitchen: stop
+                enforce_updates: true
 ```
-[living]
-    [[mpd]]
-        type = str
-        mpd_host = 127.0.0.1
-        mpd_port = 6600
-        [[[state]]]
-            type = str
-            mpd_listen = state
-            mpd_send = value  # sends the item value. e.g. sh.dev.mpd.state('play') will send 'play'
-        [[[volume]]]
-            type = num
-            mpd_listen = volume
-            mpd_send = volume
-        [[[play]]]  # any call of dev.mpd.play will send 'play'
-            type = bool
-            # knx_listen ....
-            mpd_send = play
-            enforce_updates = yes
-#       [[[time]]]
-#          type = num
-#          mpd_listen = time
-        [[[total]]]
-            type = num
-            mpd_listen = total
-        [[[percent]]]
-            type = num
-            mpd_listen = percent
-        [[[repeat]]]
-            type = bool
-            mpd_listen = repeat
-            mpd_send = repeat
-        [[[title]]]
-            type = str
-            mpd_listen = title
-        [[[album]]]
-            type = str
-            mpd_listen = album
-        [[[artist]]]
-            type = str
-            mpd_listen = artist
-        [[[name]]]
-            type = str
-            mpd_listen = name
-        [[[track]]]
-            type = str
-            mpd_listen = track
-        [[[rick]]]
-            type = bool
-            mpd_file = http://rick.net/roll.m3u
-            enforce_updates = yes
-        [[[url]]]
-            type = str
-            mpd_file = value  # plays the item value
-            enforce_updates = yes
-        [[[heavy]]]
-            type = bool
-            mpd_localplaylist = heavymetal # will play the playlist heavymetal.m3u in the MPD's playlist directory
-            enforce_updates = yes
-
-[office]
-    [[mpd2]]
-        type = str
-        mpd_host = 127.0.0.1
-        mpd_port = 6601
-        [[[state]]]
-            type = str
-            mpd_listen = state
-            mpd_send = value
-        [[[volume]]]
-            type = num
-            mpd_listen = volume
-            mpd_send = volume
-        [[[name]]]
-            type = str
-            mpd_listen = name
-        [[[track]]]
-            type = str
-            mpd_listen = track
+
+## Logging
+
+You can configure the logger to see specific outputs of the plugin. That's helpfull when setting up the plugin. To see e.g. the debug-outputs of
+the plugin you have to configure the logging.yaml file. 
+
```

Comment:
The logging should be something generic and does not need to be mentioned here. We should describe the logging in a more generic way in the documentation and I guess this is already [there](http://smarthomeng.de/v1_3/developer/logging.html).